### ISSUE TITLE
Error when accessing a page with a page model

### ISSUE THREAD

**Issue by texnixe** [gearsdigital/enhanced-toolbar-link-dialog#10]
When I try to access a page with a custom model in the Panel, I get an error `The view "default" does not exist`.

**Comment by gearsdigital** [comment]
Thank you very much for bringing this up @texnixe! Very appreciated! 😊

Could you please provide a simple example? How do you access a page with a custom model?

**Comment by texnixe** [comment]
I think this happens as soon as you create a page model for any page. In this case, the model only has a simple function that is not really that important, so I could work around this, but in other cases, that wouldn't be great.

```
<?php 

class ThemaPage extends Page
{
    public function lastUpdated()
    {
       // code here
    }
}
```

**Comment by gearsdigital** [comment]
Thanks! I’ve have guess what the problem could be. Let me fiddle a bit… I’ll come back to you 😊

**Comment by gearsdigital** [comment]
The issues was that I've defined a (obviously) unnecessary model type declaration 🤷🏻‍♂️

**Comment by gearsdigital** [comment]
Should now be fixed with [1.2.3]( https://github.com/gearsdigital/enhanced-toolbar-link-dialog/tree/v1.2.3)
